src/platform/logging/index.ts

```diff
@@ -302,6 +302,9 @@ export function logTo(logLevel: LogLevel, message: string, ...args: Arguments):
         case LogLevel.Debug:
             traceVerbose(message, ...args);
             break;
+        case LogLevel.Trace:
+            traceVerbose(message, ...args);
```

Comment:
This is what was missing, the messages were not being written
I think this needs to be cleaned up, we have too many log levels (will look into this in debt, but for now fixed the logging)